docs/user_guide/simulations.rst

```diff
@@ -74,22 +74,22 @@ Once the ``simulation`` object is created, call function ``export_ansys_json`` t
     path = "C:\\Your\\Path\\Here\\"
     json = export_ansys_json(simulation, path, ansys_tool='hfss')
 
-Performing simulations requires Ansys-scripts, which are located at ``scripts/simulations/ansys/``. Usually, it's convenient to copy this folder to the export path by calling ``copy_ansys_scripts_to_directory``::
+Performing simulations requires Ansys-scripts, which are located at `scripts/simulations/ansys/ <https://github.com/iqm-finland/KQCircuits/tree/main/klayout_package/python/scripts/simulations/ansys>`_. Usually, it's convenient to copy this folder to the export path by calling ``copy_ansys_scripts_to_directory``::
 
     copy_ansys_scripts_to_directory(path)
 
 You can create a Windows batch file for running multiple simulations in a row by calling function ``export_ansys_bat``. The first argument is a list of exported json filenames::
 
     bat = export_ansys_bat([json], path)
 
-Alternatively, you can call ``export_ansys`` to cover last three steps. This exports multiple simulations that are stored in a list, copies the Ansys-scripts into the folder, and creates the Windows batch file::
+Alternatively, you can call `export_ansys <https://github.com/iqm-finland/KQCircuits/blob/main/klayout_package/python/scripts/macros/export/export_ansys.lym>`_ to cover last three steps. This exports multiple simulations that are stored in a list, copies the Ansys-scripts into the folder, and creates the Windows batch file::
 
     bat = export_ansys([simulation], path, ansys_tool='hfss')
 
 Ansys scripts
 ^^^^^^^^^^^^^
 
-The folder ``scripts/simulations/ansys/`` contains several IronPython scripts to run simulations in Ansys Electronics Desktop. Scripts support HFSS and Q3D Extractor frameworks.
+The folder `scripts/simulations/ansys/ <https://github.com/iqm-finland/KQCircuits/tree/main/klayout_package/python/scripts/simulations/ansys>` contains several IronPython scripts to run simulations in Ansys Electronics Desktop. Scripts support HFSS and Q3D Extractor frameworks.
```

Comment:
```suggestion
The folder `scripts/simulations/ansys/ <https://github.com/iqm-finland/KQCircuits/tree/main/klayout_package/python/scripts/simulations/ansys>`_ contains several IronPython scripts to run simulations in Ansys Electronics Desktop. Scripts support HFSS and Q3D Extractor frameworks.
```
link not working because of missing underscore